src/mscorlib/src/System/Environment.cs

```diff
@@ -1239,9 +1249,9 @@ public static IDictionary GetEnvironmentVariables()
             }
         }
 
-        #if FEATURE_CORECLR
+#if FEATURE_CORECLR
```

Comment:
Nit: looks like there are a bunch of unintentional formatting changes in the file